### ISSUE TITLE
Proper CUDNN RNN initialisation

### ISSUE THREAD

**Issue by TParcollet** [speechbrain/speechbrain#104]
While discussing with @jjery2243542 we probably linked a problem of training convergence of GRU models to a bad initialisation of the weights. Indeed, PyTorch RNNs (LSTM,GRU,RNN) are initialised without respect to Glorot / He criterion (uniform with 1/n). LiGRU do not need that thanks to ReLU + BatchNorm ( alleviate saturation) but standard tanh-based RNN might saturate with an increasing number of neurons (@jjery2243542 might be experiencing this for librispeech). 

We could add an init function that applies glorot on input to hidden and orthogonal init on hidden to hidden (+ set all the biases to zero).
